modules/microshift-rhoai-verify-model-connected.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * microshift_ai/microshift-rhoai.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhoai-verify-model-connected_{context}"]
+= Verifying that your AI model is accessible
+
+Before querying the model through the API, you can check to be certain that the model is accessible and ready to provide answers based on the connected data. The following examples continue with the {ovms}.
+
+.Prerequisites
+
+* You configured the AI model-serving runtime.
+* You uploaded your AI model to {microshift-short}.
+* The {microshift-short} cluster is running.
+* You installed {oc}.
```

Comment:
{oc} is not rendered in the preview:
<img width="345" alt="image" src="https://github.com/user-attachments/assets/bcfedf0e-b636-4e9a-82b2-5b6ca4e3703f" />

Reply:
Perhaps:
```suggestion
* You installed `oc`.
```